fud2/fud-core/src/flang/ir.rs

```diff
@@ -0,0 +1,125 @@
+//! An internal representation of flang.
```

Comment:
```suggestion
//! An internal representation of fud2 plans expressed with flang.
```

To clarify what we're fundamentally doing here.